histomicstk/filters/shape/membranefilter.py

```diff
@@ -0,0 +1,74 @@
+import numpy as np
+
+from .membraneness import membraneness
+
+
+def membranefilter(im_input, sigmas, beta, c):
+    """
+    Implements a membrane-detecting filter using eigenvalue analysis of
+    scale-space blur image Hessians.
+
+    Parameters
+    ----------
+    im_input : array_like
+        M x N grayscale image. Objects to be detected are dark.
+    sigmas : array_like
+        vector of gaussian deviations (scales) at which response is to be
+        evaluated.
+    beta : double
+        Scaling property of deviation from blob. Decrease to increase
+        importance of deviation in final score. Default value = 1/2.
+    c : double
+        Scaling property of quadratic curvature magnitude. Decrease to
+        increase the importance of curvature magnitude in final score.
+        Default value = 5.
+
+    Returns
+    -------
+    Response : array_like
+        M x N intensity image of membrane filter response.
+    Thetas : array_like
+        M x N image of membrane parallel angles in radians.
+    """
+
+    # get size of input image 'I'
+    M = im_input.shape[0]
+    N = im_input.shape[1]
+
+    # convert image to double if necessary
+    if np.issubdtype(im_input.dtype, np.uint8):
+        im_input = im_input.astype('double')
+    else:
+        im_input = im_input/im_input.max()
```

Comment:
one line is missing. Normalization should be added after the floating point. Thank you!!